athenian/database_pg.go

```diff
@@ -223,6 +224,85 @@ func (db *database) AthenianToGithub(ctx context.Context, id AccountID) ([]Githu
 	return out, rows.Err()
 }
 
+func (db *database) GetInstaflowStatus(ctx context.Context, accID AccountID) (*InstaflowStatus, error) {
+	return db.getInstaflowStatus(ctx, accID)
+}
+
+func (db *database) getInstaflowStatus(ctx context.Context, accID AccountID) (*InstaflowStatus, error) {
+	row := db.db.QueryRow(
+		ctx,
+		`SELECT account_id, account_created, fetch_started, fetch_completed, consistency_started, consistency_completed, precompute_started, precompute_completed, current_status
+		FROM public.installation_progress WHERE account_id = $1`,
+		int64(accID),
+	)
+	instaflowStatus, err := scanInstaflowStatus(row)
+	if err == pgx.ErrNoRows {
+		err = ErrNotFound
+	}
+	return &instaflowStatus, err
+}
+
+func scanInstaflowStatus(sc dbs.Scanner) (InstaflowStatus, error) {
+	var (
+		accID                int64
+		accountCreated       time.Time
+		fetchStarted         time.Time
+		fetchCompleted       time.Time
+		consistencyStarted   time.Time
+		consistencyCompleted time.Time
+		precomputeStarted    time.Time
+		precomputeCompleted  time.Time
```

Comment:
does it maybe make sense to make them pointers like we do [here](https://github.com/athenianco/metadata/blob/cc50a1f8d4fe39d3a9db2b4824408033cbbc7221/jira/database/database_issue_pg.go#L373)?

Reply:
agreed